### ISSUE TITLE
Escape String

### ISSUE THREAD

**Issue by sclausen** [paralect/robomongo#890]
It would be nice, if you can paste any multiline-string in the "Edit Document" Editor and escape the selected text, maybe with a function in the context menu. Or directly paste an automatically escaped version of the text.

**Comment by navinSing** [comment]
yes :+1: some data usually contain " or ' causing issues .. problems while saving data
